### DISCUSSION TITLE
What to do about albums?

### DISCUSSION THREAD

**Discussion by TheLastGimbus** [TheLastGimbus/GooglePhotosTakeoutHelper#135] [General]
Hi guys!

As of `v3`, album detection is now very consistent and reliable. I now nicely detec which files (after nice clear list of all the files) belong to which album. Sadly, i still don't know what to do about it. Past ideas:
- make folders with symlinks to original photos inside
  I mean... thats good for linux users, maybe someone would actually want this - if you are this person, please let know
- spit out `.json` with info about this
  That seems the best for now, and it's what `v2` has been doing
- just put them (duplicated) in separate folders
  That's some way about it...

Please let me know/vote below what options to implement :100:

EDIT: Resolved: #187

**Comment by isabelpothos** [comment]
Just putting them (duplicated) in separate album folders would be great! I have a ton of named albums that i want to keep so that would be super helpful. Thank you so much for this tool by the way, it's looks super helpful! I haven't actually gotten to use it yet, still working on unzipping and merging my photo archives.
